src/test/java/org/jvnet/hudson/plugins/platformlabeler/PlatformDetailsTaskLsbReleaseTest.java

```diff
@@ -62,16 +62,15 @@ public void testComputeLabelsForOsRelease() throws Exception {
     File lsbReleaseFile = new File(resource.toURI());
     assertTrue("File not found " + lsbReleaseFile, lsbReleaseFile.exists());
     LsbRelease release = new LsbRelease(lsbReleaseFile);
-    HashSet<String> result = details.computeLabels("amd64", "linux", "xyzzy-abc", release);
+    PlatformDetails result = details.computeLabels("amd64", "linux", "xyzzy-abc", release);
+    assertThat(result.getArchitecture(), equalTo(expectedArch));
```

Comment:
The code elsewhere in this plugin has preferred `is` rather than `equalTo`.  Since this is comparing strings, could you switch it to use `is` instead of `equalTo`?
```suggestion
    assertThat(result.getArchitecture(), equalTo(expectedArch));
```

Reply:
done